content/projects/data-science-specific/data-visualisation/_index.md

```diff
@@ -5,22 +5,46 @@ ready: True
 
 ## Tutorials
 
-Go through the DataCamp course [Introduction to Data Visualisation with
-Python](https://www.datacamp.com/courses/introduction-to-data-visualization-with-python), or if you want to fast-track, complete Kaggle's [Data Visualisation: From Non-Coder to Coder](https://www.kaggle.com/learn/data-visualization-from-non-coder-to-coder)
+**Compulsory**: Go through the DataCamp course [Introduction to Data Visualisation with
+Python](https://www.datacamp.com/courses/introduction-to-data-visualization-with-python).
```

Comment:
could this stuff go in a topic instead? It would be nice to have it as find-able as possible. If someone asks me for links to learn data visualisation then the first place I could look would be inside the topics directory.